src/database.ts

```diff
@@ -206,24 +212,48 @@ export async function getPNIDProfileJSONByPID(pid: number): Promise<PNIDProfile
 }
 
 export async function getServerByGameServerID(gameServerID: string, accessMode: string): Promise<HydratedServerDocument | null> {
-	return await Server.findOne({
-		game_server_id: gameServerID,
-		access_mode: accessMode
-	});
+	const searchModes = accessModeOrder[accessMode] || ['prod'];

```

Comment:
We want to use the `??` operator which communicates the intention clearly as it's specifically for null/undefined coalescing

We also want to use the `accessModeOrder.prod` so that the default is tied to the order object 

```suggestion
	const searchModes = accessModeOrder[accessMode] ?? accessModeOrder.prod;
```

Reply:
Fixed!